[2025-07-30T10:14:52.760+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-30T10:14:52.815+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-30T10:14:35.776293+00:00 [queued]>
[2025-07-30T10:14:52.831+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-30T10:14:35.776293+00:00 [queued]>
[2025-07-30T10:14:52.832+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-30T10:14:53.013+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-30 10:14:35.776293+00:00
[2025-07-30T10:14:53.026+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'load_data', 'manual__2025-07-30T10:14:35.776293+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpl13wbo4d']
[2025-07-30T10:14:53.030+0000] {standard_task_runner.py:91} INFO - Job 212: Subtask load_data
[2025-07-30T10:14:53.033+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=368) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-30T10:14:53.034+0000] {standard_task_runner.py:63} INFO - Started process 370 to run task
[2025-07-30T10:14:53.124+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.load_data manual__2025-07-30T10:14:35.776293+00:00 [running]> on host 06b1742b327c
[2025-07-30T10:14:53.252+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T10:14:35.776293+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T10:14:35.776293+00:00'
[2025-07-30T10:14:53.257+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-30T10:14:53.649+0000] {logging_mixin.py:188} INFO - Data loaded into PostgreSQL 'transactions' table.
[2025-07-30T10:14:53.652+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-30T10:14:53.655+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-30T10:14:53.685+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=load_data, execution_date=20250730T101435, start_date=20250730T101452, end_date=20250730T101453
[2025-07-30T10:14:53.775+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-30T10:14:53.815+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-30T10:14:53.819+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
